README.md

```diff
@@ -0,0 +1,197 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h1><b>Motorbike Front End</b></h1>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Motorbike front end] <a name="about-project"></a>
+
+**[Motorbike front end]** is a react front end application that allow a user to book an appointment to try a motorcycle.
+
+[Link to Back-End Project](https://github.com/SaneMyburg/motorbike-back-end)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="">React.js</a></li>
+    <li><a href="">Javascript</a></li>
+    <li><a href="">CSS</a></li>
+    <li><a href="">Redux</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="">N/A</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="">N/A</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **[User login]**
+- **[Add motorbike]**
+- **[Reserve motorbike]**
+- **[Delete motorbike]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+  Visual Studio Code
+  Internet
+  Node.js 
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd motorbike-front-end
+  git clone https://github.com/SaneMyburg/motorbike-front-end.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd motorbike-front-end
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm run test
+```
```

Comment:
- [ ] When I try to run the tests according to the instructions, this happens:

![image](https://github.com/SaneMyburg/motorbike-front-end/assets/57421823/072516f9-d64b-4e29-9daf-63d5d0f6142c)

This is because the only test that is found is the one that comes with react, which is failing:

![image](https://github.com/SaneMyburg/motorbike-front-end/assets/57421823/4bf397cc-c867-49ee-ac3c-65b07dfc666e)

Kindly, either implement the test or remove this file `App.test.js` if not using it to prevent confusion.

Also, please either add some frontend tests, or remove this readme instructions if you are not including them. 👍